<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ê•∑‰Ωì</family>
    <pointsize>18</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Â≠¶ÁîüÁªºÂêà‰ø°ÊÅØÁÆ°ÁêÜÁ≥ªÁªüv8.2 By L_STUDIO_EVO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Qrc/20240822_Mine1.qrc">
    <normaloff>:/new/prefix1/X:/PS/Photos/L_S_E/L_S_E-1_1.jpg</normaloff>:/new/prefix1/X:/PS/Photos/L_S_E/L_S_E-1_1.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="LB_Intro">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>20</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Â≠¶ÁîüÁªºÂêà‰ø°ÊÅØÁÆ°ÁêÜÁ≥ªÁªüv8.2 By L_STUDIO_EVO</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_LSE">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/new/prefix1/X:/PS/Photos/L_S_E/L_S_E-1_1.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Save">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>20</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>‰øùÂ≠òÂπ∂ÈÄÄÂá∫</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="PB1_Add">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Â¢ûÂä†‚ûï</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB2_Alter">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>‰øÆÊîπüñä</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB3_Delete">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Âà†Èô§‚ùå</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="LW_1">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="movement">
         <enum>QListView::Movement::Static</enum>
        </property>
        <property name="flow">
         <enum>QListView::Flow::TopToBottom</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>false</bool>
        </property>
        <property name="resizeMode">
         <enum>QListView::ResizeMode::Fixed</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::LayoutMode::SinglePass</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ViewMode::ListMode</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>ÂßìÂêç</string>
         </property>
         <property name="font">
          <font>
           <family>Ê•∑‰Ωì</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SN</string>
         </property>
         <property name="font">
          <font>
           <family>Ê•∑‰Ωì</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ËØ≠Êñá</string>
         </property>
         <property name="font">
          <font>
           <family>Ê•∑‰Ωì</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Êï∞Â≠¶</string>
         </property>
         <property name="font">
          <font>
           <family>Ê•∑‰Ωì</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ëã±ËØ≠</string>
         </property>
         <property name="font">
          <font>
           <family>Ê•∑‰Ωì</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ÊÄªÂàÜ</string>
         </property>
         <property name="font">
          <font>
           <family>Ê•∑‰Ωì</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE1_Search">
        <property name="minimumSize">
         <size>
          <width>390</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>420</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="placeholderText">
         <string>‚ÜêÈÄâÊã©ÊêúÁ¥¢È°π,Ëá™Âä®ÊêúÁ¥¢ËæìÂÖ•ÁöÑÂÜÖÂÆπ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="TV_1">
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>425</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>475</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ê•∑‰Ωì</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="L1_EventLabel">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>‰∫ã‰ª∂‚Üí</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="L2_EventTime">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="L3_Event">
        <property name="minimumSize">
         <size>
          <width>370</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>420</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>ËøûÊé•Êï∞ÊçÆÂ∫ìÊàêÂäü!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="L4_TimeLabel">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>ÂΩìÂâçÊó∂Èó¥‚Üí</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="L5_CurrentTime">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ê•∑‰Ωì</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Qrc/20240822_Mine1.qrc"/>
 </resources>
 <connections/>
</ui>
